### ISSUE TITLE
Add COVID Pipeline vs Universal Pipeline logic to the Data Dashboard Redirection

### ISSUE THREAD

**Issue by chris-kuryak** [CDCgov/prime-reportstream#12265]
## Problem statement
As we onboard more receivers to the UP, they are still being shown the Data Dashboard page. But the Data Dashboard page does not work with data sent through the UP, and therefore any receivers that are connected to the UP should be shown the Daily Data page instead of Data Dashboard.

## What you need to know
- Topics that are being used in BE
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMFBjQVE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--ef1db02e1b4fce467a7c2671e464a325a6680b1a/Screenshot%202023-11-15%20at%203.45.17%E2%80%AFPM.png)

- Example User Workflow Issue:
  - I am part of CA-DPH
  - CA-DPH has several receiver services
  - Some of those receivers services are connected to UP, some are connected to the COVID pipeline
  - The UP receiver services do not work with Data Dashboard, only COVID pipeline services work on the Data Dashboard
  - So, if I log in as a CA-DPH rep, not all of my receiver services will work on Data Dashboard, even though I can select them. I have no way of knowing when/how to access Daily Data to see the services that are working on the UP.

## Expected Response
- If an organization has any receivers connected to the UP, or is part of the list of existing receivers we are currently redirected to the Daily Data page, we should redirect them to the Daily Data page instead of Data Dashboard.

## Possible Solution
If you are part of an organization that has AT LEAST one receiver service active on the UP, we just show you Daily Data. This is determined by the “Topic” of the receiver. Additionally, if you fail that test, AND you are a part of one of these XY special organizations (special list), you also see Daily Data.

In other words, check to see if ANY receiver of an organization is connected to topics other than COVID-19 or MPOX, then show them Daily Data. Otherwise, show them Data Dashboard.

## Acceptance criteria

- [ ] Receiver Org that has any receiver services connected to the UP is redirected to the Daily Data page instead of Data Dashboard.
- [ ] If a Receiver Org is already being redirected to the Daily Data page, they should continue to be redirected to the Daily Data page instead of Data Dashboard.

**Comment by chris-kuryak** [comment]
Easiest solution may just be to hardcode this in to our existing logic as we migrate receivers over to the UP. This also ensures that we are not doing work that may be negated by rectifying our two dashboards next quarter.

**Comment by chris-kuryak** [comment]
Closing as a long term (backend) solution is no longer necessary since the two dashboards will likely not coexist in the second half of the year. So the short term solution should be fine until then.
